tests/data/1-simple-default/contentType-td-valid.jsonld

```diff
@@ -5,39 +5,48 @@
     "title": "MyCameraThing",
     "$title": "contentType",
     "description": "Thing description for a camera thing",
-    "$description": "The contentType member is used to assign a media type [RFC2046] including media type parameters as attribute-value pairs separated by a semicolon (";").",
+    "$description": "The contentType member is used to assign a media type [RFC2046] including media type parameters as attribute-value pairs separated by a semicolon (';').",
```

Comment:
This is a bare bones guess at an example. Someone more authoritative may want to correct my guess.
```suggestion
    "$description": "The contentType member is used to assign a media type [RFC2046] including media type parameters as attribute-value pairs separated by a semicolon (';'), e.g., 'attr1=val1;attr2=val2;'.",
```

Reply:
@TallTed the example can be helpful and is correct.